### ISSUE TITLE
feature: Open to collaborate feature

### ISSUE THREAD

**Issue by bdougie** [open-sauced/insights#645]
### Type of feature

Select the type of feature request, the lowercase should also be the PR prefix.

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

### Current behavior

Add a setting to the [User Settings Panel](https://github.com/open-sauced/insights/issues/382) that identifies if a user is open to collaborate. 

### Suggested solution
<img width="940" alt="Screen Shot 2022-12-07 at 3 18 07 PM" src="https://user-images.githubusercontent.com/5713670/206318340-95f1f367-87ec-447e-bafa-463042cf5e13.png">

The User Profile has a button to collaborate, but this should also add an orange circle to [user profiles](https://github.com/open-sauced/insights/issues/634) in the contributor chart and cards.

This is a mock and not perfect (note the orange should be the standard opensauced orange).

![Group 1](https://user-images.githubusercontent.com/5713670/206319939-de6e68a0-2363-4a11-bbd2-01fc07e3e130.png)

### Code of Conduct

By submitting this issue, you agree to follow our [Code of Conduct](https://docs.opensauced.pizza/contributing/code-of-conduct/).

- [ ] I agree to follow this project's Code of Conduct

### Contributing Docs

If you plan on contributing code please read - [Contribution Guide](https://docs.opensauced.pizza/contributing/introduction-to-contributing/)

- [ ]  I agree to follow this project's Contribution Docs

**Comment by TAKANOME-DEV** [comment]
Hi @bdougie, I can't find this page :disappointed_relieved: Do you mind giving me some details about this?

**Comment by bdougie** [comment]
Yeah, [Campsite](https://www.campsite.design/) is set to private by default. We might coordinate getting @open-sauced/triage team members access. I will follow up via discord. Details are still be figured out but, there will need to be API updates. This feature is also blocked by a few other issues. @SunGoldTech is currently leading these efforts.

You can follow the [milestones](https://github.com/open-sauced/insights/milestone/11) and [project board](https://github.com/orgs/open-sauced/projects/6/views/3).

**Comment by bdougie** [comment]
Example from - Idea is making a new sub nav on the user profile.

<img width="755" alt="Screen Shot 2023-01-13 at 1 57 50 PM" src="https://user-images.githubusercontent.com/5713670/212427023-6e517370-b60b-4680-9021-f67411dff693.png">

**Comment by bdougie** [comment]
This is really a duplicate of https://github.com/open-sauced/insights/issues/719 now
